plugins/auth-backend/src/providers/oidc/provider.ts

```diff
@@ -123,22 +136,13 @@ export class OidcAuthProvider implements OAuthHandlers {
       throw new Error('Refresh failed');
     }
     const profile = await client.userinfo(tokenset.access_token);
-
-    return this.populateIdentity({
-      providerInfo: {
-        accessToken: tokenset.access_token,
-        refreshToken: tokenset.refresh_token,
-        expiresInSeconds: tokenset.expires_in,
-        idToken: tokenset.id_token,
-        scope: tokenset.scope || '',
-      },
-      profile,
-    });
+    return this.handleResult({ tokenset, userinfo: profile });
   }
 
   private async setupStrategy(options: Options): Promise<OidcImpl> {
     const issuer = await Issuer.discover(options.metadataUrl);
     const client = new issuer.Client({
+      access_type: 'offline', // this option must be passed to provider to receive a refresh token
```

Comment:
@Rugvip I've updated the client options too, without passing this option, we don't get a refresh token :)

Reply:
Yep that should be fine 👍